rpcs3/Emu/CPU/CPUThread.cpp

```diff
@@ -283,6 +283,26 @@ void cpu_thread::operator()()
 		thread_ctrl::set_native_priority(-1);
 	}
 
+	if (id_type() == 2)
+	{
+		// disable denormals for SPU threads
+		volatile __m128 a;
```

Comment:
only the first load should be volatile, which should be saved at a non-volatile destination var.

Reply:
fixed

Reply:
can simply use `(const volatile __m128& )_mm_castsi128_ps(_mm_set1_epi32(0x1fc00000))` instead of volatile var.

Reply:
consider adding these babies to types.h:

```
template <typename T>
volatile T& as_volatile(T& t)
{
	return const_cast<volatile T&>(t);
}

template <typename T>
const volatile T& as_volatile(const T& t)
{
	return const_cast<const volatile T&>(t);
}
```
usage: `as_volatile(0x1fc00000)`

Reply:
fixed & pushed

Reply:
Absolutely useless thing in types.h. Bad advices.

Reply:
simplified inline ...